### PR TITLE
[WIP] Fix/Added escape helper to view output

### PR DESCRIPTION
**Pull request by ins0**
Escape all these things

### CODE REVIEW

File: module/Application/view/application/index/index.phtml [hunk 1]
```diff
@@ -26,13 +26,13 @@
                         <div class="span6 module-author">
                             <div class="row-fluid">
                                 <div class="span3">
-                                    <img src="<?php echo $module->getPhotoUrl()?>" alt="<?php echo $module->getOwner()?>" class="avatar">
+                                    <img src="<?php echo $module->getPhotoUrl()?>" alt="<?php echo $this->escapeHtml($module->getOwner()); ?>" class="avatar">
```
Comment (GeeH) [inline module/Application/view/application/index/index.phtml]: Use htmlAttr escaper on the getPhotoUrl please

File: module/Application/view/application/index/index.phtml [hunk 2]
```diff
@@ -26,13 +26,13 @@
...
                                 </div>
                                 <div class="span9">
-                                    <strong><?php echo $module->getOwner()?></strong>
+                                    <strong><?php echo $this->escapeHtml($module->getOwner()); ?></strong>
                                     <p>
                                         <span class="author-label">Github:</span>
-                                        <a href="<?php echo $module->getUrl()?>"><?php echo $module->getName()?></a>
+                                        <a href="<?php echo $module->getUrl()?>"><?php echo $this->escapeHtml($module->getName()); ?></a>
```
Comment (GeeH) [inline module/Application/view/application/index/index.phtml]: As above

File: module/Application/view/application/search/index.phtml [hunk 3]
```diff
@@ -1,14 +1,14 @@
-<?php foreach($results as $module) {?>
+<?php foreach($results as $module): ?>
 <div class="row-fluid">
     <div class="span12">
         <div class="span2">
-            <img src="<?php echo $module->getPhotoUrl()?>" alt="<?php echo $module->getName()?>" class="avatar">
+            <img src="<?php echo $module->getPhotoUrl()?>" alt="<?php echo $this->escapeHtml($module->getName()); ?>" class="avatar">
```
Comment (GeeH) [inline module/Application/view/application/search/index.phtml]: As above

File: module/Application/view/layout/layout.phtml [hunk 4]
```diff
@@ -42,11 +42,11 @@
                 ?>" title="My profile">
                 <img src="<?php
                     echo $this->zfcUserIdentity()->getPhotoUrl();
-                ?>" alt="<?php echo $this->zfcUserDisplayName();
+                ?>" alt="<?php echo $this->escapeHtml($this->zfcUserDisplayName());
                 ?>" style="width:23px;height:23px;" />
                 </a></div><a class="login" href="<?php
```
Comment (GeeH) [inline module/Application/view/layout/layout.phtml]: Use escapeUrl here please

File: module/ZfModule/view/zf-module/helper/module-description.phtml [hunk 5]
```diff
@@ -5,13 +5,13 @@
                 <div class="span5 module-author">
                     <div class="row-fluid">
                         <div class="span2">
-                            <img src="<?php echo $module['photo_url']?>" alt="<?php echo $module['name']?>" />
+                            <img src="<?php echo $module['photo_url']?>" alt="<?php echo $this->escapeHtml($module['name']); ?>" />
```
Comment (GeeH) [inline module/ZfModule/view/zf-module/helper/module-description.phtml]: escapeHtmlAttr here

File: module/ZfModule/view/zf-module/helper/module-description.phtml [hunk 6]
```diff
@@ -5,13 +5,13 @@
...
                         </div>
                         <div class="span10">
-                            <strong><?php echo $module['owner']?></strong>
+                            <strong><?php echo $this->escapeHtml($module['owner']); ?></strong>
                             <p>
                                 <span class="author-label">Github:</span>
-                                <a href="<?php echo $module['url']?>"><?php echo $module['name']?></a><br />
+                                <a href="<?php echo $module['url']?>"><?php echo $this->escapeHtml($module['name']); ?></a><br />
```
Comment (GeeH) [inline module/ZfModule/view/zf-module/helper/module-description.phtml]: escapeUrl here

File: module/ZfModule/view/zf-module/helper/module-view.phtml [hunk 7]
```diff
@@ -5,13 +5,13 @@
...
-                            <img src="<?php echo $module['photo_url']?>" alt="<?php echo $module['name']?>" />
+                            <img src="<?php echo $module['photo_url']?>" alt="<?php echo $this->escapeHtml($module['name']); ?>" />
                         </div>
                         <div class="span10">
-                            <strong><?php echo $module['owner']?></strong>
+                            <strong><?php echo $this->escapeHtml($module['owner']); ?></strong>
                             <p>
                                 <span class="author-label">Github:</span>
-                                <a href="<?php echo $module['url']?>"><?php echo $module['name']?></a><br />
+                                <a href="<?php echo $module['url']?>"><?php echo $this->escapeHtml($module['name']); ?></a><br />
```
Comment (GeeH) [inline module/ZfModule/view/zf-module/helper/module-view.phtml]: escapeUrl here

File: module/ZfModule/view/zf-module/index/view.phtml [hunk 8]
```diff
@@ -5,7 +5,7 @@
```
Comment (GeeH) [inline module/ZfModule/view/zf-module/index/view.phtml]: IIRC the meta data in head tags need to be sanitized also

Comment (GeeH) [inline module/ZfModule/view/zf-module/index/view.phtml]: IIRC the meta data in head tags need to be sanitized also

**Comment by ins0** [comment]
didn't notice this helper LMAO, thanks @GeeH will update this

**Comment by GeeH** [comment]
Thank you very much @ins0 :)

**Pull request merged**